### ISSUE TITLE
New feature: Build monitor view support

### ISSUE THREAD

**Issue by kuldazbraslav** [jimbydamonk/jenkins-job-builder-addons#4]
Hello, it would be great if the addons support Build Monitor view as well. We use it as view for managers because it's so simple that everyone can understand it.

**Comment by khurrammalik** [comment]
I agree with @kuldazbraslav, do you think it can be added as a feature request. Really looking forward for this add-on. Great work guys on building such a useful tool.

**Comment by RudolfVonKrugstein** [comment]
+1, I would love that too.

**Comment by RudolfVonKrugstein** [comment]
Is there enough demand that this will come? Or should I expect that this will not come in the near future?

**Comment by kuldazbraslav** [comment]
Actually, as I was missing many features in JJB, I switched to JobDSL: https://jenkinsci.github.io/job-dsl-plugin/
